rs-react-app/src/pages/_test_/MainPage.test.tsx

```diff
@@ -0,0 +1,199 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+import { render, screen, fireEvent, act } from '@testing-library/react';
+import { BrowserRouter, useSearchParams } from 'react-router-dom';
+import MainPage from '../MainPage';
+import type { PokemonDetails } from '../../api/pokeapi';
```

Comment:
❗️Using relative paths like '../../api/pokeapi' makes the code harder to read and maintain.
Would be great to set up import aliases (e.g. @api/pokeapi) via tsconfig.json for cleaner imports.
You can google: “TypeScript path aliases tsconfig” if needed.